### PR TITLE
Updated neighbourhood names

### PR DESCRIPTION
**Pull request by stepps00**
This PR corrects a discrepancy with the English names and removes incorrect name translations in other `name:*` properties. No PIP work, this can be merged once approved.

### CODE REVIEW

**Comment by stepps00** [comment]
These incorrect names seem to have snuck in as part of the most recent Wikipedia names run. The English names seem to be a relic of Zetashapes, though I don't have the git history to confirm..

File: data/420/552/245/420552245.geojson [hunk 1]
```diff
@@ -21,38 +21,12 @@
...
-    ],
-    "name:heb_x_preferred":[
-        "\u05db\u05d9\u05db\u05e8 \u05e4\u05e8\u05e9\u05e8\u05df"
-    ],
-    "name:jpn_x_preferred":[
-        "\u30d7\u30ec\u30b7\u30a7\u30fc\u30ec\u30f3\u5e83\u5834"
-    ],
-    "name:nld_x_preferred":[
-        "Pre\u0161erenplein"
+        "St Marys Park"
```
Comment (nvkelso) [inline data/420/552/245/420552245.geojson:25]: This should match the `wof:name`, which has a `.` after the `St`. So this value should be `St. Marys Park`... though perhaps the Marys should be possessive? Place names are weird, please research and confirm.

**Review by nvkelso — CHANGES_REQUESTED** [review]
See comment

File: data/420/552/245/420552245.geojson (hunk 1 above)
Comment (stepps00) [reply data/420/552/245/420552245.geojson:25]: I see a mix of both "St. Marys Park" and "St. Mary's Park":

https://www.trulia.com/n/ca/san-francisco/st-marys-park/155139/
https://www.niche.com/places-to-live/n/st-marys-park-san-francisco-ca/
https://www.zillow.com/st.-marys-park-san-francisco-ca/
https://www.nytimes.com/2011/12/11/us/st-marys-park-in-san-francisco.html
http://www.stmaryspark.org/

Though most non-real estate sources use the apostrophe, so I've updated to match.

**Review by nvkelso — APPROVED** [review]

**Pull request merged**